components/Card.js

```diff
@@ -6,63 +6,107 @@ import {
   Button,
   Image,
   TouchableOpacity,
+  TextInput,
+  Keyboard,
+  TouchableWithoutFeedback,
 } from "react-native";
+
 import files from "../constants/files";
+import style from "../constants/style";
+
 const Card = (props) => {
   const [sheets, setSheets] = useState(0);
+  const [sheetIcon, setSheetIcon] = useState(files.images.sheet1);
+
+  const updateSheetIconHandler = () => {
+    if (sheets >= 50) {
+      setSheetIcon(files.images.sheet4);
+    } else if (sheets >= 25) {
+      setSheetIcon(files.images.sheet3);
+    } else if (sheets >= 10) {
+      setSheetIcon(files.images.sheet2);
+    } else {
+      setSheetIcon(files.images.sheet1);
+    }
+  };
   const incrementSheetHandler = (props) => {
     setSheets(sheets + 1);
+    updateSheetIconHandler();
+    Keyboard.dismiss();
   };
   const decrementSheetHandler = (props) => {
     if (sheets <= 0) {
       return;
     }
     setSheets(sheets - 1);
+    updateSheetIconHandler();
+    Keyboard.dismiss();
+  };
+  const changeSheetCountHandler = (props) => {
+    if (props !== "") {
```

Comment:
can there be a case where props are undefined?
Please review your logic for this function